scipy/stats/tests/test_stats.py

```diff
@@ -1177,11 +1177,17 @@ def test_structured_arrays(self):
 
 
 class TestMode(TestCase):
+    def test_empty(self):
+        data1 = []
+        vals = stats.mode(data1)
```

Comment:
I was going for consistency with the rest of the tests, but I agree it is clearer the way you suggest. I have made the change.